src/app/Models/Order.php

```diff
@@ -96,4 +96,16 @@ public function scopeShowOrder($query, $id_order)
         ->where('orders.id',$id_order);
     }
 
+    public function scopeGetCountOrders($query) {
+        return $query->selectRaw('COUNT(*) as count_orders')
```

Comment:
Vì sao dùng COUNT(*)?
Nếu chỉ COUNT(id) thì có đựoc không?

Reply:
Count(*) sẽ get tất cả hàng chưa cả null, nhưng id là khoá chính sẽ là not null thì chắc chắn sẽ không có trường nào không có id cả. Nên theo em dùng count(*) hay count('id') đều như nhau ạ.